rel-eng/tito.props

```diff
@@ -74,6 +76,7 @@ whitelist = foreman-installer
   rubygem-rkerberos
   rubygem-rubyipmi
   rubygem-thor
+  rubygem-openscap
```

Comment:
Still incorrect?  The other tags are right, it just needs removing from this line.